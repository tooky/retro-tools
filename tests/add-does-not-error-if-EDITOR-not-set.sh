EDITOR= retro_date=1997-06-05 retro add
ls doc/retro/*/*/*
cat doc/retro/1997/06/05.md
