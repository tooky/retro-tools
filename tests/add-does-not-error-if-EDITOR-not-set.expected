EDITOR= retro_date=1997-06-05 retro add
doc/retro/1997/06/05.md
ls doc/retro/*/*/*
doc/retro/1997/06/05.md
cat doc/retro/1997/06/05.md
# 1997-06-05
## Who

## What happened

## Insights

## Actions
